testing-and-tdd/testing-and-tdd.md

```diff
@@ -135,11 +135,11 @@ Sadly, these go by even more names than listed but, in general, these are the hi
 
 #### Load Testing
 
-We won't go into these much here, but load testing is essential testing that your code can handle an increased amount of input or activity. Lets say you have a website that works when you have one user, what happens when you have 10,000 users? If you have a database that is working with three rows, what happens when you have thousands? You want to make sure that you program still works reliably regardless of user volume.
+We won't go into these much here, but load testing is essential testing that your code can handle an increased amount of input or activity. Lets say you have a website that works when you have one user, what happens when you have 10,000 users? If you have a database that is working with three rows, what happens when you have thousands? You want to make sure that your program still works reliably regardless of user volume.
```

Comment:
@LeiaQuesada, in addition to your correction to this section, I believe this:

> ...but load testing is essential testing that...

could be changed to:

...but load testing is essentially testing that...